src/vaadin-date-picker-mixin.html

```diff
@@ -399,6 +400,14 @@
         '';
     }
 
+    _openedChanged(opened) {
+      if (!opened) {
+        return;
+      }
+
+      this._updateAlignmentAndPosition();
```

Comment:
Isn’t line 346 redundant (and even a harmful for performance) after this (calling the `_updateAlignmentAndPosition()` method inside the `open()` method?

Reply:
You are right! Thanks, I've removed that line.